metrics/healthcheck.go

```diff
@@ -1,6 +1,6 @@
 package metrics
 
-// Healthchecks hold an error value describing an arbitrary up/down status.
+// Healthcheck methods hold an error value describing an arbitrary up/down status.
```

Comment:
Please don't add `methods` all over, it's a simple interface.

Reply:
Thank you - updated.